p2p/dandelion-tower/src/pool/incoming_tx.rs

```diff
@@ -0,0 +1,112 @@
+//! Contains [`IncomingTx`] and [`IncomingTxBuilder`]
+use crate::{State, TxState};
+
+/// An incoming transaction that has gone through the preprocessing stage.
+pub struct IncomingTx<Tx, TxId, PId> {
```

Comment:
> peer is never abbreviated as P...

generics are often very short so this should not matter.

> This would trigger the lint

True but that's more an issue with the lint, I'll change it anyway